app/[locale]/dashboard/components/import/config/platforms.tsx

```
@@ -59,18 +64,21 @@ type StepComponent =
   name: string
   description: string
-  category: 'Direct Account Sync' | 'Custom CSV Import' | 'Platform CSV Import'
+  category: 'Direct Account Sync' | 'Intelligent Import' | 'Platform CSV Import'
   videoUrl?: string
   details: string
   logo: {
-    path: string
-    alt: string
+    path?: string
+    alt?: string
+    component?: ComponentType<{}>
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)